homeassistant/components/here_travel_time/sensor.py

```diff
@@ -150,79 +120,36 @@ async def async_setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the HERE travel time platform."""
-    api_key = config[CONF_API_KEY]
-    here_client = herepy.RoutingApi(api_key)
-
-    if not await hass.async_add_executor_job(
-        _are_valid_client_credentials, here_client
-    ):
-        _LOGGER.error(
-            "Invalid credentials. This error is returned if the specified token was invalid or no contract could be found for this token"
+    hass.async_create_task(
```

Comment:
Typically configuration.yaml imports happen from the setup/async_setup function in init.py. Can this move there?

Reply:
Wouldn't that mean that the configuration.yaml which is currently looking like this:

```yaml
sensor:
  - platform: here_travel_time
    name: travel_time_work_from_home
[...]
```

would have to look like this:

```yaml
here_travel_time:
  - name: travel_time_work_from_home
[...]
```

Which would be a breaking change?

Reply:
Ah, sorry I missed that.